src/snowflake/snowpark/_internal/udf_utils.py

```diff
@@ -22,7 +22,7 @@
     get_type_hints,
```

Comment:
This file will be handled in https://github.com/snowflakedb/snowpark-python/pull/1245?